store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java

```diff
@@ -1714,8 +1714,7 @@ private boolean isSpaceToDelete() {
             }
 
             {
-                String storePathLogics = StorePathConfigHelper
-                    .getStorePathConsumeQueue(DefaultMessageStore.this.getMessageStoreConfig().getStorePathRootDir());
+                String storePathLogics = DefaultMessageStore.this.getStorePathLogic();
```

Comment:
This logic is the same, right?

Reply:
Yes, the logical path is the same, here is just a unified method of obtaining the path.

Reply:
> This logic is the same, right?

Yes, the logical path is the same, here is just a unified method of obtaining the path.